apps/system_message/resource.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+
+
+class SystemMessagesResource(Resource):
+    schema = {
+        "is_active": {"type": "boolean", "default": False},
+        "type": {
+            "type": "string",
+            "allowed": ["warning", "alert", "primary/info", "success"],
+            "required": True,
+        },
+        "message_title": {"type": "string", "required": True},
+        "message": {"type": "string", "required": True},
+        "user_id": {"type": "string", "required": True},
```

Comment:
I think it should be rather `Resource.rel("user", required=True)` so it will validate the value